tutorials/concepts/arch.md

```diff
@@ -0,0 +1,21 @@
++++
+title = "Landscape Architecture"
+weight = 4
+template = "doc.html"
++++
+
+# Landscape Architecture
+
+Landscape is designed using a particular userspace architecture model known as store/hook/thread. This model divides applications into multiple parts, two gall applications and threads.
```

Comment:
> Landscape is designed using a particular userspace architecture model known as store/hook/thread.

The name of the model is "store/hook/thread"? Or is that what the model consists of?

> This model divides applications into multiple parts, two gall applications and threads.

This sentence doesn't parse in my head. Are the multiple parts the gall applications and threads? Or are there multiple parts, two gall applications, and threads, or is "gall applications and threads" a single thing or should there be an Oxford comma?

Reply:
> The name of the model is "store/hook/thread"? Or is that what the model consists of?

Those are the parts idk if it has a name that doesn't suck.

Reply:
but it's sort of the same as like Model View Controller so this would be SHT (lol)

Reply:
Maybe just say that the model consists of stores, hooks, and threads, explained below.